### DISCUSSION TITLE
WebView2 InsecurePrivateNetworkRequests

### DISCUSSION THREAD

**Discussion by fysmart** [MicrosoftEdge/WebView2Feedback#3936] [Q&A]
https://a.com send a http request to local server: https://b.local has a CORS error. 
see this: https://github.com/WICG/private-network-access/issues/67

Local sever need set the http response header: Access-Control-Allow-Private-Network: true
but i can't configure this local server and webview2 **WebResourceResponseReceived** event can not change **e.Response**.

so.. how to make this option to **Disabled** use **WebView2 AdditionalBrowserArguments**
or other solutions?
![image](https://user-images.githubusercontent.com/17706457/153176684-844a55a0-90be-49ed-8c26-e540e6e53fa3.png)

**Comment by weiwei11245** [comment]
Hey, I think your one is very similar to mine. You can have a look at mine
https://github.com/MicrosoftEdge/WebView2Feedback/issues/1950

**Comment by fysmart** [comment]
> Hey, I think your one is very similar to mine. You can have a look at mine #1950

Thank you, but it's not work-_-

![(%J~@9}SBV%5`M5{9FPK06R](https://user-images.githubusercontent.com/17706457/153502215-f59f298a-51e0-40ed-8494-b3c97cd2012f.png)
![QQ截图20220211054814](https://user-images.githubusercontent.com/17706457/153502221-c805b441-f832-43bd-b742-13c489b658ee.jpg)
